citadel/indico_citadel/search.py

```diff
@@ -63,7 +64,7 @@ def search(self, query, user=None, page=1, object_types=(), **params):
             **filter_query
         }
         # Filter by the objects that can be viewed by users/groups in the `access` argument
-        if access := get_user_access(user):
+        if access := get_user_access(user) if not admin_override_enabled else ['IndicoAdmin']:
```

Comment:
I'm not sure I like the way this line looks like now :D 

I'd either move it to a separate statement (instead of inlining it with `:=`), or pass `admin_override_enabled` as an arg to `get_user_access` and check it in here (could even include the `user.is_admin`  check there just to be completely on the safe side)

Reply:
Yes, I was feeling the same. I initially made it into a different line without `:=`. I think the second option is better.